### PR TITLE
More descrpitive platformer comments

### PR DESCRIPTION
**Pull request by AhmetAffan0**
# What issue is this addressing?

## What this PR does | solves
I fixed them.

### CODE REVIEW

**Comment by hajimehoshi** [comment]
I'm confused. Why did you make a new PR?

File: examples/platformer/main.go [hunk 1]
```diff
@@ -127,7 +128,7 @@ func (g *Game) Update() error {
 		g.gopher = &char{x: 50 * unit, y: groundY * unit}
 	}
 
-	// Controls
+	// Controls.
```
Comment (hajimehoshi) [inline examples/platformer/main.go:131]: This is a word, not a sentence. Remove `.`

Comment (AhmetAffan0) [reply examples/platformer/main.go:131]: Isn't it supposed to be like that? I thought the changes I made in the other branch were not gone. I don't know if the changes in the previous branch were visible, so I created a new one.

**Comment by AhmetAffan0** [comment]
> I'm confused. Why did you make a new PR?

Isn't it supposed to be like that? I thought the changes I made in the other branch were not gone. I don't know if the changes in the previous branch were visible, so I created a new one.

**Comment by hajimehoshi** [comment]
> Isn't it supposed to be like that? I thought the changes I made in the other branch were not gone. I don't know if the changes in the previous branch were visible, so I created a new one.

No it is not. Adding a new commit onto the same branch without creating a new branch/PR is fine.

File: examples/platformer/main.go [hunk 2]
```diff
@@ -113,7 +113,7 @@ func (c *char) draw(screen *ebiten.Image) {
 	}
 
 	op := &ebiten.DrawImageOptions{}
-	// Use a smaller scale than 1 to shrink the image.
+	// Use a smaller scale than 1 to shrink the image. For make image bigger use more than 1 to make bigger.
```
Comment (hajimehoshi) [inline examples/platformer/main.go:116]:
```go
// Use a smaller scale than 1 to shrink the image.
```

**Review by hajimehoshi — APPROVED** [review]
LGTM, thanks!

**Pull request merged**